src/client/public/scripts/dash/outgoing.ts

```diff
@@ -0,0 +1,35 @@
+import {asyncAppend} from '../../../../../node_modules/lit-html/lib/async-append.js';
+import {html, render} from '../../../../../node_modules/lit-html/lit-html.js';
+import * as api from '../../../../types/api';
+
+import {prTemplate} from './prs';
+
+async function* getNextElement(userLogin: string|null) {
```

Comment:
Might be worth adding a comment how this works with lit. I'm not familiar with generators and the use of `function*` and `yield` is new to me.